packages/mobile/src/fiatExchanges/FiatExchangeAmount.tsx

```diff
@@ -272,6 +272,7 @@ function FiatExchangeAmount({ route }: Props) {
             }`}</Text>
           </View>
           <TextInput
+            multiline={true}
```

Comment:
Is it needed to have the input be multiline?

Reply:
This was to prevent the text from being cropped when users input amount over 1000 with large accessibility font on iOS; The input should remain on one line unless it would overflow.